system/Images/Handlers/BaseHandler.php

```diff
@@ -361,12 +361,12 @@ public function rotate(float $angle)
 	{
 		// Allowed rotation values
 		$degs = [
-			90,
-			180,
-			270,
+			(float) 90,
+			(float) 180,
+			(float) 270,
```

Comment:
Why not just write these as `90.0`, `180.0`, `270.0`?

Reply:
because we can pass "90" as float ref https://3v4l.org/BFb9u

Reply:
I don't understand the difference. Isn't `(float) 90 === 90.0`?

Reply:
oh, you're correct.  I will update to `90.0`, not in front of pc right now

Reply:
fixed.